atmosphere/settings/local.py.j2

```diff
@@ -397,6 +397,14 @@ if has_logstash and LOGSTASH_HOST:
                                      message_type='atmo-deploy', version=1)
     threepio.deploy_logger.addHandler(fh)
 
+
+# NEW RELIC
+try:
+    from atmosphere.settings import new_relic
+    NEW_RELIC_CONFIGURED = True
+except ImportError:
+    NEW_RELIC_CONFIGURED = False
+
```

Comment:
I think we'd need to bring the settings variables from `atmosphere.settings.new_relic` into the `atmosphere.settings` context here or in `atmosphere/settings/__init__.py`. 

Because for atmosphere, we use the `NEW_RELIC_ENVIRONMENT` string in the [`wsgi.py`](https://github.com/cyverse/atmosphere/blob/master/atmosphere/wsgi.py) start up code:
- https://github.com/cyverse/atmosphere/blob/master/atmosphere/wsgi.py#L38-L51

For Troposphere, the template to inject the browser-agent would also be in the `.new_relic`.

Maybe I'm misunderstanding the scoping for the Python namespace here.